packages/shared/src/hooks/supabase/mutations/useFeedbackMutation.ts

```
@@ -1,7 +1,6 @@
 import { FeedbackService } from '@src/utils';
 import { useMutation } from '@tanstack/react-query';
-
-import useSupabaseFeedbackClientQuery from './useSupabaseFeedbackClientQuery';
+import { useSupabaseFeedbackClientQuery } from '../queries';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)